conf/messages

```diff
@@ -41,6 +41,9 @@ common.stateBenefits = State benefits
 common.view = View
 common.yes = Yes
 
+common.error.amountMaxLimit = Enter an amount less than £100,000,000,000
+common.error.invalid_currency_format = Enter the amount in the correct format
```

Comment:
Why in `snake_case`? Honest question, not sure if there is any reason that justifies it? Otherwise I'd align it with the rest as `lowerCamelCase`

Reply:
thats how the default is in form, if i changed it in one lace, i would like to do it in all the reops